testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/atom/AtomComplexModelTest.java

```diff
@@ -83,6 +82,13 @@
                 AtomComplexModelState.class,
                 AtomComplexModelUuid.class,
                 AtomComplexModelVersionNumber.class);
+
+        war.addAsManifestResource(PermissionUtil.createPermissionsXmlAsset(
```

Comment:
With Ingo's fix in this PR https://github.com/resteasy/Resteasy/pull/1126 (related to AbstractJAXBContextFinder and JAXBContextWrapper) it reveals subsequent issue with:
`at org.jboss.resteasy.plugins.providers.atom.Entry.getAnyOtherJAXBObject(Entry.java:366)
`
This should maybe get another doPrivilege block in the code, not permission in the testcase.